explainaboard/analysis/feature.py

```diff
@@ -39,19 +39,23 @@ def __init__(
         description: str | None = None,
         func: Callable[..., Any] | None = None,
         require_training_set: bool | None = None,
+        skippable: Optional[bool] = False,
     ) -> None:
         """Initializes FeatureType object.
 
         Args:
             description: Description of this feature.
             func: Function to calculate this feature from other features.
             require_training_set: Whether this feature relies on the training samples.
+            skippable: set it to True if this feature is optional in the output file
+                or it does not require default bucket analysis
```

Comment:
We don't need to provide external contexts.
```suggestion
            skippable: set it to True if this feature is optional.
```